src/main.js

```diff
@@ -27,19 +37,39 @@
     }
 
     attachNonPreventedListener(document, "click", (e) => {
+        // ignore ctrl / meta clicks (may be opening in new tab or window)
+        if (e.ctrlKey || e.metaKey) {
+            return;
+        }
+		
         const body = document.body;
-
         for (var el = e.target; el && el !== body; el = el.parentNode) {
             if (el.nodeName.toLowerCase() === "a") {
+                // ignore links with a .no-ajaxy class
+                if (el.classList.contains('no-ajaxy')) {
```

Comment:
You don’t need this class. Add standards based target=“_self” instead

Reply:
Other ajaxify scripts usually have no-ajaxy as a class to avoid weird conflicts when a link is already being managed by another script. For example an image gallery script I'm aware of might have been surrounding <img> tags with an <a> tag. Since your script isn't binding click handlers to individual elements generated <a> tags may break existing scripts.